### PR TITLE
Use library from crayfish-commons

### PR DESCRIPTION
**Pull request by jonathangreen**
## GitHub Issue

Issue:
https://github.com/Islandora-CLAW/CLAW/issues/577

Related to:
https://github.com/Islandora-CLAW/Crayfish-Commons/pull/2

## What does this Pull Request do?

Modifies Hypercube to use the class added to Crayfish-Commons here:
https://github.com/Islandora-CLAW/Crayfish-Commons/pull/2

# How should this be tested?

- Get a JWT. I added a dsm here and then performed a write operation on a FedoraResource to print it to the page.
- Use an HTTP client like Postman, making sure to add your token as an Authorization header for both of the following requests.
- Add a tiff to Fedora as a NonRdfSource at http://localhost:8080/fcrepo/rest/some/crazy/path
- Run the builtin webserver with php -S localhost:8088 -t /path/to/Hypercube/src
- Request the OCR for the tiff sending a GET request to Hypercube at http://localhost:8088/some/crazy/path

# Interested parties
@Islandora-CLAW/committers @dannylamb

### CODE REVIEW

**Comment by dannylamb** [comment]
Nice, this looks good too.  We could just let this subsume #18 and call it a day with both of them.

We're still gonna be blocked by Islandora-CLAW/CLAW#585.  As soon as I'm done setting up my next slew of PRs for the d8 configuration to do images in islandora and islandora_image, I can take a stab at it.

File: Hypercube/src/app.php [hunk 1]
```diff
@@ -13,10 +15,18 @@
 $app->register(new ServiceControllerServiceProvider());
 $app['hypercube.controller'] = function () use ($config) {
     return new HypercubeController(
-        new TesseractService($config['executable'])
+        new CmdExecuteService(),
```
Comment (dannylamb) [inline Hypercube/src/app.php]: Needs logger.  Maybe we can feed it a monolog service using http://silex.sensiolabs.org/doc/2.0/providers/monolog.html?  I believe it conforms to the PSR logging interface.

Comment (jonathangreen) [inline Hypercube/src/app.php]: We can inject a logger, but we don't have to. Its set to a default:
https://github.com/Islandora-CLAW/Crayfish-Commons/blob/master/src/CmdExecuteService.php#L24

Comment (jonathangreen) [inline Hypercube/src/app.php]: If we do end up wanting to use a logger, we can use the silex monolog implementation like we are doing in Houdini:
https://github.com/Islandora-CLAW/Crayfish/pull/20/files#diff-63757cc0ace6867bee89a08c62fd76f2R17

**Comment by jonathangreen** [comment]
Just rebased this on top of #18, should be good to go now.

**Pull request merged**